### ISSUE TITLE
Issues with URL parameters

### ISSUE THREAD

**Issue by jensscherbl** [symphonycms/symphony-2#2484]
Looks like URL parameters (`/data/params/url-fieldname`) behave differently (and not as supposed to) in `2.6.x` as they did in `2.5.x`.

If I submit a form with multiple checkboxes named `example[]`, I get 

    <url-example.1>value</url-example.1>
    <url-example.2>value</url-example.2>
    <url-example.3>value</url-example.3>

instead of

    <url-example>
        <item>value</item>
        <item>value</item>
        <item>value</item>
    </url-example>

Can someone confirm this?

**Comment by brendo** [comment]
Nah, it's always been the first way. There was talk of moving to the second approach but it hasn'tt happened as both scenarios are valid. See #1807

**Comment by jensscherbl** [comment]
Ah ok. Guess it has been customized by @designermonkey then.

**Comment by jensscherbl** [comment]
@brendo How can I filter a data source with the values of my `example` checkboxes then?

**Comment by jensscherbl** [comment]
@brendo By the way, I think we SHOULD move to the second approach. As all the issues and discussions point out, this use case is far more common. It would also be consistent with data source output parameters. Any chance to reconsider this for `3.0.0`?

**Comment by nitriques** [comment]
> By the way, I think we SHOULD move to the second approach. 

Totally!!! +1000

**Comment by jensscherbl** [comment]
@nitriques This would be a breaking change.

**Comment by nitriques** [comment]
Thanks Jens!
